bindings/pulp/bindings/server_info.py

```diff
@@ -51,12 +52,3 @@ def get_distributors(self):
         """
         path = self.base_path + 'distributors/'
         return self.server.GET(path)
-
-    def ping(self):
-        """
-        Retrieves basic status information from the server.
-
-        @return: Response
-        """
-        path = '/v2/services/status/'
-        return self.server.GET(path)
```

Comment:
`DeprecationWarning` and `PendingDeprecationWarning` are ignored by default in Python 2.7+ unless the user adds `-Wd` to the command line.

I will add a bug to remove this in 3.0 and mention the bug and deprecation in the method's docblock.